tests/integration/test_publish_on_linkedin.py

```diff
@@ -0,0 +1,254 @@
+# Standard Library
+from unittest import mock
+
+# Third Party Stuff
+import pytest
+import requests
+from django.conf import settings
+from rest_framework import status
+from tests import factories as f
+
+# nexus Stuff
+from nexus.base import exceptions as exc
+from nexus.social_media import services, tasks
+from nexus.social_media.models import Post
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def base_data(client):
+    post = f.create_post(image=None)
+    api_url_ugc = f"{settings.LINKEDIN_API_URL_BASE}ugcPosts"
+    linkedin = settings.LINKEDIN_AUTH
```

Comment:
I would rename this variable to `linkedin_auth` to signify what it is actually doing.